doc/source/ray-contribute/development.rst

```diff
@@ -71,35 +71,45 @@ Every time you want to update your local version you can pull the changes from t
 Prepare the Python environment
 ------------------------------
 
-.. tip::
+You probably want to have some type of Python virtual environment. For example, you could use Anaconda's ``conda``. 
 
-  To setup an isolated Anaconda environment, see :ref:`ray_anaconda`.
+.. tabbed:: conda
 
-You will probably want to have some type of Python virtual environment. For example you could use Python's integrated ``venv`` module to create one:
+    Set up an isolated Anaconda environment using the instructions in :ref:`ray_anaconda`.
 
-.. code-block:: shell
+    Creating a new virtual environment can come with older versions of ``pip`` and ``wheel``. To avoid problems when you install packages, use the module ``pip`` to install the latest version of ``pip`` (itself) and ``wheel``:
```

Comment:
@pcmoritz Are these instructions needed for the `conda` option?

Reply:
Typically pip is pretty up to date for the conda distribution, I think we can skip this for conda :)